.github/CODEOWNERS

```diff
@@ -15,3 +15,7 @@
 
 # Jenkinsfile changes should be reviewed by Release Engineering
 #Jenkinsfile @daos-stack/release-engineering
+
+# any PR that touches client API or high level client code
+#src/client @daos-stack/client-api-watcher
```

Comment:
Is it your intent not to enable this yet?

Reply:
i saw everything is disabled, so didn't know if i should start enabling this now.